pkg/cmdutils/config.go

```diff
@@ -121,33 +120,19 @@ func (c *ClusterConfig) addBKFlags(flags *pflag.FlagSet) {
 	flags.StringVar(
 		&c.BKWebServiceURL,
 		"bookie-service-url",
-		bookkeeper.DefaultWebServiceURL,
+		c.BKWebServiceURL,
```

Comment:
This change is for consistency with other flags.  Has the effect of preferred flags, then config, then default values.